core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -526,6 +526,15 @@ oppia.factory('explorationCorrectnessFeedbackService', [
       return (typeof value === 'boolean');
     };
 
+    child.isCorrectnessFeedbackEnabled = function() {
```

Comment:
(FYI, for contrast, note that it's fine for this to start with "is", since it is a function.)